src/main/java/by/custom/utilcalculator/exception/InvalidOrderAnswerException.java

```diff
@@ -0,0 +1,24 @@
+package by.custom.utilcalculator.exception;
+
+import by.custom.utilcalculator.exception.constants.UtilsborErrorCode;
+import by.custom.utilcalculator.exception.constants.UtilsborErrorDescription;
+
+public class InvalidOrderAnswerException extends UtilsborException {
```

Comment:
I think `InvalidOrderCommandException` is better. It's better to avoid extending business language if it's possible.

Reply:
Yes, it's better. Done